flashlight/nn/modules/Module.cpp

```diff
@@ -16,60 +16,111 @@
 
 #include <stdexcept>
 
+#include "flashlight/common/Logging.h"
 #include "flashlight/nn/modules/Module.h"
 
-
 #include "flashlight/nn/Init.h"
 
 namespace fl {
 
 Module::Module() = default;
 
-Module::Module(const std::vector<Variable>& params)
```

Comment:
It would be good to also keep this one and just have the keys (strings) to be e.g. "layer1, layer2...".